src/printer/tptp/tptp_printer.cpp

```diff
@@ -91,6 +82,210 @@ void TptpPrinter::toStream(std::ostream& out, const UnsatCore& core) const
+{
+  printUnknownCommand(out, "define-fun");
+}
+
+void TptpPrinter::toStreamCmdDefineNamedFunction(
+    std::ostream& out,
+    const std::string& id,
+    const std::vector<Node>& formals,
+    TypeNode range,
+    Node formula) const
+{
+  printUnknownCommand(out, "define-named-function");
+}
+
+void TptpPrinter::toStreamCmdCheckSat(std::ostream& out) const
+{
+  printUnknownCommand(out, "check-sat");
+}
+
+void TptpPrinter::toStreamCmdCheckSat(std::ostream& out, Node n) const
+{
+  printUnknownCommand(out, "check-sat");
+}
+
+void TptpPrinter::toStreamCmdCheckSatAssuming(std::ostream& out,
+                                              std::vector<Node> nodes) const
+{
+  printUnknownCommand(out, "check-sat-assuming");
+}
+
+void TptpPrinter::toStreamCmdQuery(std::ostream& out, Node n) const
+{
+  printUnknownCommand(out, "query");
+}
+
+void TptpPrinter::toStreamCmdSimplify(std::ostream& out, Node n) const
+{
+  printUnknownCommand(out, "simplify");
+}
+
+void TptpPrinter::toStreamCmdGetValue(std::ostream& out,
+                                      const std::vector<Node>& nodes) const
+{
+  printUnknownCommand(out, "get-value");
+}
+
+void TptpPrinter::toStreamCmdGetAssignment(std::ostream& out) const
+{
+  printUnknownCommand(out, "get-assignment");
+}
+
+void TptpPrinter::toStreamCmdGetModel(std::ostream& out) const
+{
+  printUnknownCommand(out, "ge-model");
+}
+
+void TptpPrinter::toStreamCmdGetProof(std::ostream& out) const
+{
+  printUnknownCommand(out, "get-proof");
+}
+
+void TptpPrinter::toStreamCmdGetUnsatCore(std::ostream& out) const
+{
+  printUnknownCommand(out, "get-unsat-core");
+}
+
+void TptpPrinter::toStreamCmdGetAssertions(std::ostream& out) const
+{
+  printUnknownCommand(out, "get-assertions");
+}
+
+void TptpPrinter::toStreamCmdSetBenchmarkStatus(std::ostream& out,
+                                                BenchmarkStatus status) const
+{
+  printUnknownCommand(out, "set-info");
+}
+
+void TptpPrinter::toStreamCmdSetBenchmarkLogic(std::ostream& out,
+                                               const std::string& logic) const
+{
+  printUnknownCommand(out, "set-logic");
+}
+
+void TptpPrinter::toStreamCmdSetInfo(std::ostream& out,
+                                     const std::string& flag,
+                                     SExpr sexpr) const
+{
+  printUnknownCommand(out, "set-info");
+}
+
+void TptpPrinter::toStreamCmdGetInfo(std::ostream& out,
+                                     const std::string& flag) const
+{
+  printUnknownCommand(out, "get-info");
+}
+
+void TptpPrinter::toStreamCmdSetOption(std::ostream& out,
+                                       const std::string& flag,
+                                       SExpr sexpr) const
+{
+  printUnknownCommand(out, "set-option");
+}
+
+void TptpPrinter::toStreamCmdGetOption(std::ostream& out,
+                                       const std::string& flag) const
+{
+  printUnknownCommand(out, "get-option");
```

Comment:
Can you omit any overrides that just print unknown?

Reply:
Done!